.github/build/action.yml

```diff
@@ -0,0 +1,76 @@
+name: 'Build'
+inputs:
+  build_identifier:
+    description: 'If specified will be added to the version in order to identify non official releases'
+    required: false
+    default: ''
+outputs:
+  changelog_outputs_path:
```

Comment:
Maybe we'll give these nice names in the future, e.g. changelog, archive, checksum

Reply:
should be done